tests/test_cube.py

```diff
@@ -1,6 +1,29 @@
+import os
+
+import numpy as np
+
 from cubehandler import Cube
 
+this_dir = os.path.dirname(os.path.abspath(__file__))
+
 
 def test_empty_cube():
     cube = Cube()
     assert cube is not None
+
+
+def test_read_cube():
+    cube = Cube.from_file(this_dir + "/CH4_HOMO.cube")
```

Comment:
if you use pathlib, this becomes:
```suggestion
    cube = Cube.from_file(this_dir / "CH4_HOMO.cube")
```